src/components/career-form/career-components.tsx

```diff
@@ -150,16 +149,17 @@ export const Actions = ({
   isSubmitting,
   fieldErrors,
 }: ActionsProps) => {
+  const { t } = useTranslation();
   if (error) {
     return (
       <>
         <FormError error={error} />
-        <p>
```

Comment:
Yes except a different tag is given

Reply:
that's what I meant. `p` transports a specific semantic "being a paragraph" but I think there is not downsides, not even from a11y side. I was just asking as we should not do the same with headlines for example because this would have an impact on different levels 🙏